### PR TITLE
FUSETOOLS-2342: Fix issues in NewTransformationWizard found by SonarQube

### PR DESCRIPTION
**Pull request by jpav**

### CODE REVIEW

File: transformation/chrysalix/chrysalix-common/.project [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (apupier) [inline transformation/chrysalix/chrysalix-common/.project]: this file should not be part of the commit

File: transformation/chrysalix/chrysalix-designer/.project [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (apupier) [inline transformation/chrysalix/chrysalix-designer/.project]: this file should not be part of the commit

File: transformation/chrysalix/chrysalix-runtime/.project [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (apupier) [inline transformation/chrysalix/chrysalix-runtime/.project]: this file should not be part of the commit

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java [hunk 4]
```diff
@@ -73,10 +70,10 @@
 import org.jboss.tools.fuse.transformation.editor.internal.wizards.XMLPage;
 import org.jboss.tools.fuse.transformation.editor.internal.wizards.XformWizardPage;
 import org.jboss.tools.fuse.transformation.extensions.DozerConfigContentTypeDescriber;
-
 import com.sun.codemodel.JCodeModel;
 import com.sun.codemodel.JDefinedClass;
 import com.sun.codemodel.JPackage;
+import javafx.scene.shape.Path;
```
Comment (apupier) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java]: this import seems strange. It was org.eclipse.core.runtime.Path before

Comment (jpav) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java]: Crap, that's happening because of something screwed up in my local env.  I was hoping I could make these changes without it affecting anything, but that obviously is failing.  I'll deal with this tomorrow and update.

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java [hunk 5]
```diff
@@ -85,6 +83,7 @@
 
     public static final String CAMEL_CONFIG_PATH = MavenUtils.RESOURCES_PATH + "META-INF/spring/camel-context.xml"; //$NON-NLS-1$
     private static final String OBJECT_FACTORY_NAME = "ObjectFactory"; //$NON-NLS-1$
+    private static final String CAMEL_PKG = "org.apache.camel"; //$NON-NLS-1$
```
Comment (apupier) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java]: Optional: what do you think to call it CAMEL_GROUPID ? if I understand well the code, this will be more accurate.
